v8 release (#89)
PediatricOpenTargets/OpenPBTA-analysis: doc/release-notes.md

      - `sample_id` and `aliquot_id` for some TARGET samples were miscoded before. This release fixed the issue per [ticket 145](https://github.com/PediatricOpenTargets/ticket-tracker/issues/145)
      - `broad_histology` for TARGET samples were updated as following: `Acute Lymphoblastic Leukemia` and `Acute Myeloid Leukemia` were merged to `Hematologic malignancies`; `Clear cell sarcoma of the kidney`, `Rhabdoid tumor`, and `Wilms tumor` were combined as `Renal tumors`; `Osteosarcoma` is changed to `Mesenchymal, non-meningothelial tumors` and `Neuroblastoma` is converted to `Embryonal tumors`. See [ticket 136](https://github.com/PediatricOpenTargets/ticket-tracker/issues/136)
      - For `gtex_group == "Cells"`, the `composition` column is changed from `Solid Tissue` to `Derived Cell Line` per discussion in d3b center [ticket 43](https://github.com/d3b-center/D3b-codes/pull/43)

>       - For `short_histology`, `NBL` is converted to `Neuroblastoma` to be consistent with other samples 

Reviewer: ```suggestion
      - For `short_histology`,  neuroblastoma samples previously annotated as `NBL` or `Embryonal tumor` are converted to `Neuroblastoma` to be consistent with other samples 
```

Reviewer: For some reason, I cannot change line 8 in release notes:

- deiltaed changes: 
should be - `detailed changes:`

Author: Fixed!

      - Updated MB subtypes in the histologies file per [ticket 148](https://github.com/PediatricOpenTargets/ticket-tracker/issues/148)
      - Some GMKF WGS samples does not have `germline_sex_estimate` as indicated in [ticket 168](https://github.com/PediatricOpenTargets/ticket-tracker/issues/168). Added using the file in the discussion of [ticket 159](https://github.com/PediatricOpenTargets/ticket-tracker/issues/159)

>       - Some TARGET WXS samples miss `tumor_ploidy` that are actually avaialble - and those samples now have updated ploidy using the file in the discussion session of [ticket 160](https://github.com/PediatricOpenTargets/ticket-tracker/issues/160)

Reviewer: ```suggestion
      - Some TARGET WXS samples miss `tumor_ploidy` that are actually available - and those samples now have updated ploidy using the file in the discussion session of [ticket 160](https://github.com/PediatricOpenTargets/ticket-tracker/issues/160)
```

      - cnv-cnvkit.seg.gz [ticket 156](https://github.com/PediatricOpenTargets/ticket-tracker/issues/156)
      - cnv-controlfreec.tsv.gz [ticket 156](https://github.com/PediatricOpenTargets/ticket-tracker/issues/156)
      - snv-consensus-plus-hotspots.maf.tsv.gz [ticket 156](https://github.com/PediatricOpenTargets/ticket-tracker/issues/156)

>     - Update method to call CNV consensus as described in [ticket 134](https://github.com/PediatricOpenTargets/ticket-tracker/issues/134) and [ticket 149](https://github.com/PediatricOpenTargets/ticket-tracker/issues/149). Briefly, CNV called by `manta` were filtered to contain only `filter == "PASS"` before going into the consensus calling workflow. And in the subsequent step, instead of ony retaining CNV calls that have 50% reciprocal overlap between callers (which was too stringent), the criteria is expanded to include small CNV regions that are 90% covered by a larger CNV. Additionally, WGS and WXS samples now went through different CNV calling pipeline. CNV consensus is only called for WGS samples and hence consensus cnv file only contain WGS sampels [ticket 152](https://github.com/PediatricOpenTargets/ticket-tracker/issues/152): 

Reviewer: ```suggestion
    - Update method to call CNV consensus (WGS) as described in [ticket 134](https://github.com/PediatricOpenTargets/ticket-tracker/issues/134) and [ticket 149](https://github.com/PediatricOpenTargets/ticket-tracker/issues/149). Briefly, CNV called by `MantaSV` were filtered to contain only `filter == "PASS"` before going into the consensus calling workflow. In the subsequent step, instead of only retaining CNV calls that have 50% reciprocal overlap between callers (which was too stringent), the criteria is expanded to include small CNV regions that are 90% covered by a larger CNV. The consensus is the overlapping region. 
```

for the future, please write one sentence per line for easier review :)  

I removed that samples went through two pipelines bc we had that in v7 as well and our filenames/data files description indicate such.

Author: Gotcha! Will do!

      - cnv_consensus_seg.gz (WGS samples only - in S3 bucket s3://kf-openaccess-us-east-1-prd-pbta/open-targets/v8/ but not in md5sum.txt file)

>     -As a result of changing consensus calling criteria and adding TARGET WXS DNA sample results to `cnv-cnvkit.seg.gz` and `cnv-controlfreec.tsv.gz`, the following files were updated:

Reviewer: ```suggestion
    -As a result of changing consensus calling criteria and adding new TARGET WXS DNA sample results to `cnv-cnvkit.seg.gz` and `cnv-controlfreec.tsv.gz`, the following files were updated:
```

      - consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz [ticket 159](https://github.com/PediatricOpenTargets/ticket-tracker/issues/159) and [ticket 160](https://github.com/PediatricOpenTargets/ticket-tracker/issues/160) 

>       - consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz [ticket 159](https://github.com/PediatricOpenTargets/ticket-tracker/issues/159) and [ticket 160](https://github.com/PediatricOpenTargets/ticket-tracker/issues/160)

Reviewer: ```suggestion
      - consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz [ticket 159](https://github.com/PediatricOpenTargets/ticket-tracker/issues/159) and [ticket 160](https://github.com/PediatricOpenTargets/ticket-tracker/issues/160) 
      - consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz [ticket 159](https://github.com/PediatricOpenTargets/ticket-tracker/issues/159) and [ticket 160](https://github.com/PediatricOpenTargets/ticket-tracker/issues/160)
```
You can remove these individual files from the md5sum as well, leave on s3, since we have a combined file.

      
    - Added `consensus_wgs_plus_cnvkit_wxs.tsv.gz` which is a merge of `consensus_wgs_plus_cnvkit_wxs_autosomes.tsv.gz` and `consensus_wgs_plus_cnvkit_wxs_x_and_y.tsv.gz` per [ticket 161](https://github.com/PediatricOpenTargets/ticket-tracker/issues/161)
    - Updated `ensg-hugo-rmtl-mapping.tsv` file per [ticket 146](https://github.com/PediatricOpenTargets/ticket-tracker/issues/146). The previous release of this file does not contain all gene ENSG IDs and symbols that are present in `snv-consensus-plus-hotspots.maf.tsv.gz`. This update merged GENCODE V28 and V38 to allow inclusion of more gene ENSG IDs and symbols. 

>     - Futher update `ensg-hugo-rmtl-mapping.tsv` [PR 48 D3b codes](https://github.com/d3b-center/D3b-codes/pull/48)

Reviewer 2: ```suggestion
    - Futher update `ensg-hugo-rmtl-mapping.tsv` [PR 48 D3b codes](https://github.com/d3b-center/D3b-codes/pull/48) to include all gene ENSG ID to symbol mappings in v7 `ensg-hugo-rmtl-mapping.tsv`.
```

Thank you for updating the release notes. Just adding a suggestion to clarify this update.

Author: @logstar, thanks so much! I will implement your suggested changes. One quick question - now that ticket 135 is in, do you want to add `all cohort` independent samples in the release as well? If so, which ones?

Reviewer 2: Thank you for the question. Yes. It would be great to have `all-cohorts` independent sample lists in the release as well. All corresponding ones for `each-cohort` independent sample lists in slack message <https://opentargetspediatrics.slack.com/archives/C02B5K2B99C/p1629165953031500?thread_ts=1629158109.022400&cid=C02B5K2B99C> should be sufficient. If not, we could add more in v9.
